cloudinit/distros/ug_util.py

```diff
@@ -107,181 +90,149 @@ def _normalize_users(u_cfg, def_user_cfg=None):
                 if u and u not in users:
                     users[u] = {}
         elif isinstance(user_config, dict):
+            n = 'default'
```

Comment:
Nit: I think these 3 lines could just be
```python
n = user_config.pop('name', 'default')
```

Reply:
Thanks, good catch.